pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/TopicPoliciesTest.java

```diff
@@ -3015,6 +3019,63 @@ public void testLoopCreateAndDeleteTopicPolicies() throws Exception {
             });
         }
     }
+
+    @Test
+    public void testGetTopicPoliciesWithCleanCache()
```

Comment:
@coderzc Can you reproduce the issue with this test with the fix? I have tried it on my laptop without this fix. But can't reproduce the issue. The fix looks good, just the test can't protect the changes of this PR.

Reply:
I have rewritten the test to make it can reproduce the issue.

Reply:
@codelipenghui PTAL.